libfwupd/fwupd-enums.h

```diff
@@ -101,6 +102,7 @@ typedef enum {
 #define FWUPD_DEVICE_FLAG_NEEDS_BOOTLOADER	(1u << 6)	/* Since: 0.7.3 */
 #define FWUPD_DEVICE_FLAG_REGISTERED		(1u << 7)	/* Since: 0.9.7 */
 #define FWUPD_DEVICE_FLAG_NEEDS_REBOOT		(1u << 8)	/* Since: 0.9.7 */
+#define FWUPD_DEVICE_FLAG_WAITING_FOR_REPLUG	(1u << 9)	/* Since: 1.0.2 */
```

Comment:
This didn't make 1.0.2, so I guess bump to 1.0.3.

Reply:
Agree, fixed, thanks.